packages/jest-emotion/src/utils.js

```diff
@@ -189,7 +198,9 @@ export function getStylesFromClassNames(
 }
 
 export function getStyleElements(): Array<HTMLStyleElement> {
-  let elements = Array.from(document.querySelectorAll('style[data-emotion]'))
+  let elements = global.document
```

Comment:
I think `global` might not be always available - so maybe better to use a simple check of `typeof document !== 'undefined'`

Reply:
also - when this check matters? for the enzyme's case when tests can be run without jsdom available globally?

Reply:
that's right, this code should always be run in node, but the code shouldn't simply fail if JSDOM is not installed.